### PR TITLE
Add 'Managing Supply' section to custodial accounts page

### PR DESCRIPTION
**Pull request by JakeUrban**
Updates the 'Setup a Custodial Account' page to include more information about supporting non-native assets.

cc @lydiat

### CODE REVIEW

File: content/docs/building-apps/setup-custodial-account.mdx [hunk 1]
```diff
@@ -169,22 +176,31 @@ function onPooledPayment(customerId, destination, amount, asset) {
...
+
+_Note_: Users cannot send a non-native asset to your pooled account (nor receive one from your customers) unless they have opted into that asset by opening a trustline to its issuer.
 
 </Alert>
 
-Finally, you'll need to adapt your balance sheet management (e.g. the internals of `depositIntoPool` and `withdrawFromPool`, [above](#code-framework)) to track things per-asset for each customer. Since this is backend-specific, we won't provide example code here.
+
+### Managing Supply
+
+Custodials usually hold a float of the assets they list on their platform. Unlike Stellar’s native asset (XLM), which can only be sourced from the Stellar Decentralized Exchange (SDEX), non-native assets can also be sourced directly from the asset’s issuer. For example, custodians can establish a Circle Account to source Stellar USDC.
```
Comment (lydiat) [inline content/docs/building-apps/setup-custodial-account.mdx:202]: Would making "Circle Account" a link (https://www.circle.com/blog/circle-account-and-api-services-now-support-stellar-usdc) be too spammy?

Comment (rice2000) [reply content/docs/building-apps/setup-custodial-account.mdx:202]: I think adding a link is acceptable.

Comment (rice2000) [inline content/docs/building-apps/setup-custodial-account.mdx:202]:
```suggestion
Custodials usually hold a float of the assets they list on their platform. Unlike Stellar’s native asset (XLM), which can only be sourced from the Stellar Decentralized Exchange (SDEX), non-native assets can also be sourced directly from the asset’s issuer. For example, custodians can establish a [Circle Account](https://www.circle.com/blog/circle-account-and-api-services-now-support-stellar-usdc) to source Stellar USDC.
```

**Review by rice2000 — APPROVED** [review]
I like @lydiat's suggestion to add a link.  I think it will be useful for readers who want to find out more.  What do you think, @JakeUrban ?  Does that work for you?

File: content/docs/building-apps/setup-custodial-account.mdx (hunk 1 above)
Comment (JakeUrban) [reply content/docs/building-apps/setup-custodial-account.mdx:202]: I think adding a link makes sense

**Pull request merged**